index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <!-- fontawesme link -->
+    <script src="https://kit.fontawesome.com/fc7e7ce012.js" crossorigin="anonymous"></script>
+    <!-- style css -->
+    <link rel="stylesheet" href="style.css">
+
+
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="navbar">
+                <ul class="top-navbar-menu">
+                    <li><a href="">Facebook</a></li>
+                    <li><a href="">Twitter</a></li>
+                    <li><a href="">English</a></li>
+                    <li><a href="">My Page</a></li>
+                    <li><a href="">Log out</a></li>
+                </ul>
+                <div class="bottom-navbar">
+                    <a class="logo" href="index.html">Logo</a>
+                    <ul class="bottom-navbar-menu">
+                        <li><a href="about.html">About</a></li>
+                        <li><a href="">Program</a></li>
+                        <li><a href="">Join</a></li>
+                        <li><a href="">Sponsor</a></li>
+                        <li><a href="">News</a></li>
+                        <li><a href="">Campiagn</a></li>
+                    </ul>   
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
```

Comment:
#### You have made a good effort to make this project to this point, well done 👏🏾 . 
- [ ] But the HTML structure of your page can be better. Since you have a `main` tag as the main container of the content of the website. Inside this main tag, semantically all the three sections, the _**Headline**_, _**Main program section**_, and the _**Featured speakers section**_ should use a `section` tag. below 👇🏾  is how you should structure your HTML to be semantically correct.
A similar thing applies to your About page HTML structure.
I wish you the best. 

```html
<main>
  <section>
    Headine section content is placed here
  </section>
  <section>
    main program content is placed here
    <div>
      <article>Lecture</article>
      <article>exhibition</article>
      <article>Forum</article>
      <article>workshop</article>
      <article>Tour</article>      
    </div>
  </section>
  <section>
    <div>
      <article>Speaker one</article>
      <article>Speaker two</article>
      <article>Speaker three</article>
      <article>Speaker four</article>
      <article>Speaker five</article>
      <article>speaker six</article>
    </div>
  </section>
  <section>
    partner section content is placed here
  </section>
  <footer>
    footer section content is placed here
  </footer>
</main>

```